libs/features/jarvis-chat.js

```diff
@@ -217,8 +218,8 @@ const startInitialization = async (config, event) => {
   }
 
   window.AdobeMessagingExperienceClient.initialize({
-    appid: config.jarvis.id,
-    appver: config.jarvis.version,
+    appid: getMetadata('jarvis-surface-id') || config.jarvis.id,
+    appver: getMetadata('jarvis-version') || config.jarvis.version,
```

Comment:
Allowing to set a Jarvis version in the metadata will lead to forgotten pages on outdated version, which will lead to broken experiences in the future.

It's not a good idea to have this configurable by GWP.

Reply:
Actually, the ticket requirement is that it should be controlled by authors in Milo just like they could control it in AEM.

Reply:
The `appver` is setting the surface version, which together with `appid` are two variables that define the Jarvis workflow and design. Ideally the same `appver` should be used, but there might be new combinations which use a different version and without being able to specify it, the Jarvis initialisation will not work.

Reply:
@mokimo - I do understand your point now, the `jarvis-version` naming is a bit misleading. @nishantka - whad do you think about using `jarvis-surface-version` ?

Reply:
Sure, I will update the property name.

Reply:
updated the version property @narcis-radu @mokimo